src/components/DownloadFileButton.vue

```diff
@@ -1,18 +1,14 @@
 <template>
   <a :href="`/api/uploads/${upload.id}/download`">
-    <img
-      class="mr-4"
-      src="/assets/img/cloud-download.svg"
-      alt=""
-      width="20"
-      height="20"
-    />
+    <button class="btn btn-primary ml-2">
```

Comment:
you don't need to embed a `button` inside an `a` anchor. instead, use button classes on the anchor itself. https://getbootstrap.com/docs/4.6/components/buttons/#button-tags

Reply:
updated to inline the button class on the anchor tag, and also removed the now unused svg file